src/controllers/auth/login.ts

```diff
@@ -0,0 +1,28 @@
+import * as Interfaces from "@interfaces";
+import * as Utils from "@utils";
+
+const login: Interfaces.Middlewares.Async = async (req, res, next) => {
+  const body: Interfaces.User = req.body;
+  try {
+    const { username, password } = body;
+    const user = await Utils.getUserByUname(username);
+    if (!user) {
+      return res.status(400).json({ msg: "No user found" });
```

Comment:
use sucesss (like in `src/globals/success/health.ts`) or errors, and set required status code

Reply:
👍🏻